src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/example-global-iam.yml

```diff
@@ -34,6 +41,41 @@ Resources:
               - "*"
       Roles:
         - adf-cloudformation-deployment-role # This role is created in the global.yml and is the default cloudformation deployment role for ADF.
+  # ADFTerraformRole:
+  #   # This is the role that will be used to deploy Terraform resources from within 
+  #   # the target account. You should scope this policy depending on what you would 
+  #   # like to deploy within certain Organizational Units.
+  #   # NOTE: below is a sample IAM policy. This policies should NOT be used for purposes other than testing.
+  #   # Uncomment this line if you want to enable the terraform extensions
+  #   Type: AWS::IAM::Role
+  #   Properties:
+  #     RoleName: "adf-terraform-role"
+  #     AssumeRolePolicyDocument:
+  #       Version: "2012-10-17"
+  #       Statement:
+  #         - Effect: Allow
+  #           Sid: "AssumeRole"
+  #           Principal:
+  #             AWS:
+  #               - !Sub arn:aws:iam::${DeploymentAccountId}:role/adf-codebuild-role
+  #           Action:
+  #             - sts:AssumeRole
+  #     Path: /
+  # ADFTerraformPolicy:
+  #   Type: AWS::IAM::Policy
+  #   Properties:
+  #     PolicyName: "adf-terraform-policy"
+  #     PolicyDocument:
+  #       Version: "2012-10-17"
+  #       Statement:
+  #         - Effect: Allow
+  #           Action:
+  #             - "ec2:*"
```

Comment:
Could you add a comment that states that this list needs to contain the permissions required to deploy the resources using Terraform. So in this example, it is allowed to deploy ec2 and s3 resources.

Actually, lets change the resources to two simple ones (ec2 is not that simple).
Let's pick a Lambda function instead. Then list the required actions specifically.

Reply:
Replace with a simpler example